### PR TITLE
Break closing paren of ConditionalExpression in member chains

### PR DESCRIPTION
**Pull request by duailibe**
Still related to #2775, maybe a better way to print this case.

```js
(a
  ? b
  : c
).prop;

(a
  ? b
  : c
).call();

(a
  ? b
  : c
)
  .call1()
  .call2();
```

If you guys think this is cool, let me know if I should do the same for LogicalExpression (I believe so) and we can do it in a single commit

Thanks!

### CODE REVIEW

File: tests/member/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 
+exports[`conditional.js 1`] = `
+(valid
+  ? helper.responseBody(this.currentUser)
+  : helper.responseBody(this.defaultUser))
+.prop;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(valid
+  ? helper.responseBody(this.currentUser)
+  : helper.responseBody(this.defaultUser)).prop;
```
Comment (suchipi) [inline tests/member/__snapshots__/jsfmt.spec.js.snap]: I understand why it's happening, but this kinda feels like a regression on the changes in https://github.com/prettier/prettier/pull/2784. I think I'm okay with it, though.

Comment (duailibe) [reply tests/member/__snapshots__/jsfmt.spec.js.snap]: Actually, that's my mistake I forgot to update this test :flushed:

**Comment by duailibe** [comment]
@suchipi I forgot to update a snapshot 😅 

I updated it with what I think would work for logical expressions

**Comment by suchipi** [comment]
Looks like CI is failing due to a format violation; could you fix?

**Comment by duailibe** [comment]
Oops so sorry about that.

Fixed!

**Pull request merged**